packages/node-red-s1seven-api/lib/certificates/notarize-certificate.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+module.exports = function (RED) {
+  const path = require('node:path');
+  require('dotenv').config({ path: path.resolve(__dirname, '../../.env') });
+
+  const {
+    exitContext,
+    setNewContext,
+  } = require('../utils/async-local-storage');
+  const { createAxiosInstance } = require('../utils/axios');
+  const {
+    getApiMode,
+    getAccessToken,
+    getCurrentCompanyId,
+  } = require('../utils/getters');
+  const requestHandler = require('../utils/requestHandler');
+  const validateCertificate = require('../utils/validateCertificate');
+
+  // eslint-disable-next-line sonarjs/cognitive-complexity
+  function notarizeCertificate(config) {
+    RED.nodes.createNode(this, config);
+    const node = this;
+    const globalContext = this.context().global;
+    const apiConfig = RED.nodes.getNode(config.apiConfig);
+
+    node.on('input', async (msg, send, cb) => {
+      function done(err) {
+        exitContext(cb, err);
+      }
+      setNewContext(apiConfig, msg);
+
+      const companyId = getCurrentCompanyId(globalContext);
+      const accessToken = getAccessToken(globalContext);
+      const mode = getApiMode(globalContext);
+
+      // request parameters
+      const identity =
+        msg.identity || config.identity || globalContext.get('identity');
+      if (!accessToken) {
+        node.warn(RED._('notarize.errors.accessToken'));
```

Comment:
The error path needs to be updated to `certificates.errors.accessToken`.

Reply:
i think i will revert the change in the translations instead as i made an attempt to centralize translations in one file that failed.